index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <title>Odin Landing Page</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">Welcome to Jaipur</div>
+      <div>home</div>
+      <div>About</div>
+      <div>Contact us</div>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-text">
+          <h1>Sign up to book your visit</h1>
+          <p>
+            Jaipur holds the distinction of being the first planned city of India. Renowned globally for its coloured gems, the capital city of Rajasthan combines the allure of its ancient history with all the advantages of a metropolis.
+          </p>
+          <button class="button intro-button">Sign up</button>
+        </div>
+        <div class="intro-image"><img src="photos/jaipur.jpeg" height="300" width="500" alt="Jaipur image"></div>
+      </div>
+      <div class="info">
+        <h2>Places to visit</h2>
+        <ul>
+          <li>
+            <div> <img src="photos/hawa mahal.jpeg" width="170" alt="Hawa mahal"> </div>
```

Comment:
Why do we need a wrapper div here, it doesn't seem to be doing anything. pls remove

also img tag is not closed, make it a self-closing tag